FirebaseSwiftUI/FirebaseFacebookSwiftUI/Sources/Services/AuthService+Facebook.swift

```diff
@@ -10,8 +10,8 @@ import FirebaseAuthSwiftUI
 public extension AuthService {
   @discardableResult
   func withFacebookSignIn(scopes scopes: [String]? = nil) -> AuthService {
-    facebookProvider = FacebookProviderAuthUI(scopes: scopes)
-    register(provider: facebookProvider!)
+    FacebookProviderAuthUI.configureSharedInstance(scopes: scopes)
+    register(provider: FacebookProviderAuthUI.shared)
```

Comment:
Switched to having a private init() for Facebook provider and making it a singleton which is kept on `shared` as there really should only be one instance of it anyway. I can roll this back if you don't like.